scripts/build/linux-cross

```diff
@@ -13,6 +13,7 @@ CROSS_OSARCH="linux/amd64 linux/arm linux/ppc64le"
 # linux/s390x
 
 echo "Building all linux binaries"
+# TODO(stevvooe): Remove gox. This just wraps environement variables.
```

Comment:
@dnephin Being able to do `make GOOS=windows` is a huge advantage. I don't see why `bash` and `parallel` is required at all. We can make a script that invokes make, if that is required.